packages/application-shell/src/components/theme-switcher/new-colors-schema-announcement-dialog.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { InfoDialog } from '@commercetools-frontend/application-components';
+import mcRebrandingAnnouncementSVG from '@commercetools-frontend/assets/images/mc-rebranding-announcement.svg';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import { STORAGE_KEYS } from '../../constants';
+import messages from './messages';
+
+const NewColoursSchemaAnnouncementDialog = () => {
+  const intl = useIntl();
+
+  const [
+    hasUserSeenNewDesignNotificationDialog,
+    setHasUserSeenNewDesignNotificationDialog,
+  ] = useState(
+    window.localStorage.getItem(
+      STORAGE_KEYS.IS_NEW_DESIGN_NOTIFICATION_CLOSED
```

Comment:
nit:
Quick note: [We've opted for](https://github.com/commercetools/ui-kit/pull/2714#issuecomment-1935576354) the term `recolouring` in the codebase instead of `rebranding` or `redesign` or `new design` and it's now in widespread use. I don't think that blocks us with proceeding with merging this PR as is

Reply:
Thank you,

updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3445/commits/cca7d527f7876017e64153390fd7b506cb4ae97d